### DISCUSSION TITLE
Tab Stacks (Sliding Panes) & Social Image Sharing

### DISCUSSION THREAD

**Discussion by mettaseva** [oleeskild/obsidian-digital-garden#130] [Ideas]
@oleeskild ~ 

(1) I'm curious: Have you given any thought to providing an "Andy Mode" (Sliding Panels) option in our published sites? (Currently called "**[tab stacks](https://www.youtube.com/watch?v=mmmJCttfA_o)**" in the newer versions of Obsidian.)

(2) Also, is there any way I can currently use an Open Graph social image sharing feature with ODG that would allow my visitors to share my images/links on various social media sites?

For example, you can see the "share on hover" social sharing option I've installed on my HTML site images **[HERE](https://mettazetty.com)** ~ and this is the **[Open Graph Generator app](https://iplocation.io/open-graph-generator)** I'm currently using.

With my thanks in advance,
Metta

**Comment by oleeskild** [comment]
1. I have thought about it. Currently it isn't something I'm prioritising, as it is would a large feature to implement. But maybe sometime in the future. 
2. With the latest version, social media image can be achieved by using the new [metatags](https://dg-docs.ole.dev/advanced/note-specific-settings/#metatags) feature.

**Reply by mettaseva** [reply]
Thank you for your follow-up and feedback, @oleeskild 

(1) I understand about the time commitment. Appreciate your willingness to consider sliding panels as an option in the future.
(2) Fabulous! Thank you so much for this new feature. I'll review and test and then let you know if I have any additional questions.

With appreciation,
Metta
